powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/ActivationService.java

```diff
@@ -264,7 +259,7 @@ public ActivationLayer1Response createActivation(ActivationLayer1Request request
                     final String userId = activationProvider.lookupUserIdForAttributes(identity, context);
 
                     // If no user was found or user ID is invalid, return an error
-                    if (userId == null || userId.equals("") || userId.length() > 255) {
+                    if (userId == null || userId.isBlank() || userId.length() > 255) {
```

Comment:
Suggestion: we can also use `StringUtils.hasText`: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html#hasText(java.lang.String)